content/en/blog/2024/prometheus-compatibility-survey/index.md

```diff
@@ -0,0 +1,147 @@
+conventions should use dots as the namespace delimiter, and underscores as the
+delimiter between "multi-word-dot-delimited components" (e.g.
+`http.response.status_code`). On the other hand, Prometheus
+[uses underscores](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels)
+as its delimiter.
+
+Currently, when exporting in Prometheus format from an OpenTelemetry SDK, all
+dots are changed to underscores to comply with the Prometheus requirements. We
+wanted to learn whether OpenTelemetry users who used these exporters preferred
+to keep the dots in the original metric name, or liked the consistency with
+existing Prometheus metrics of translating to underscores.
+
+Of users who indicated they used OpenTelemetry for metrics, and PromQL as their
+query language, 54% preferred keeping the original OpenTelemetry metric name
+including dots, and 46% want metric names that match Prometheus conventions with
+only underscores.
+
+![Dots vs underscores pie chart](dots-vs-underscores.png)
+
+When we asked about specific example PromQL queries or alerts, the results
+roughly aligned with the results above. Around 42% of users only selected
+queries with dots, and around 39% only selected queries that had underscores.
+The final 19% selected a mix of queries that included dots or underscores,
+indicating they are likely OK with either approach.
+
+## Units in Metric Names
+
+OpenTelemetry [specifies](/docs/specs/semconv/general/metrics/#units) that units
+should not generally be included in the metric name. Prometheus conventions
+[recommend](https://prometheus.io/docs/practices/naming/#metric-names) that the
+unit be included as a suffix of the metric name. OpenMetrics goes a step further
+and
+[requires this unit suffix](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#unit).
+Currently, when exporting in Prometheus format from an OpenTelemetry SDK, the
+unit is added as a suffix to the metric name.
+
+Of users who indicated they used OpenTelemetry for metrics, and PromQL as their
+query language, 37% thought units should be a required suffix for metric names,
+and 46% thought units should not be added to metric names. The final 17%
+preferred the unit in the metric name, but didn't think it should be required.
+
+![Units in metric name pie chart](units-in-metric-name.png)
+
+When we asked about specific example PromQL queries or alerts, the results were
+somewhat more favorable to including the unit in the metric name compared with
+the question above. Around 45% of users only selected queries that included the
+unit, and around 28% only selected queries that excluded the unit. The final 27%
+selected a mix of queries that included or excluded the unit, indicating they
+are likely OK with either approach.
+
+## Trends
+
+### Correlation between Unit and Delimiter Preferences
+
+Not surprisingly, preferences generally split into two groups: Those that prefer
```

Comment:
Why it's not surprising? Again, some context could help! If this could be the only possible outcome, we might as well get rid of the introductory adverb.

Reply:
added to the background.